packages/admin-panel-server/src/routes/ExportDashboardVisualisationRoute.ts

```diff
@@ -4,9 +4,20 @@
  *
  */
 
+import { Request } from 'express';
+
 import { Route } from '@tupaia/server-boilerplate';
 
-export class ExportDashboardVisualisationRoute extends Route {
+import { DashboardVisualisationObject } from '../viz-builder';
+
+export type ExportDashboardVisualisationRequest = Request<
+  Record<string, never>,
+  { contents: DashboardVisualisationObject; filePath: string; type: string },
+  { visualisation: DashboardVisualisationObject },
+  Record<string, never>
+>;
```

Comment:
Yeah! I worry it's a bit too Tupaia-centric for dev-chat. And we've been trying to make them more fun again, and I can't imagine anything less fun than getting lectured by me about typescript 😜 

Perhaps could set up a meeting for it and see who's interested?

Reply:
Haha I see what you mean, I recently deleted 3 items I had added in the dev agenda because I realised they are too dull!

> Nothing better that talking with 15 devs about ES module exports on a Friday arvo during lockdown!! 🥳 🍷 👯
> _kael89, 3/9/2021_

Perhaps even a message in a Slack channel pointing to a PR will do...